app/javascript/app/components/sectors-agriculture/drivers-of-emissions/card-pie-chart/card-pie-chart-component.jsx

```diff
@@ -0,0 +1,131 @@
+import React, { PureComponent } from 'react';
+import { Card, PieChart, Tag, Loading, NoContent } from 'cw-components';
+import PropTypes from 'prop-types';
+import { TabletLandscape, TabletPortraitOnly } from 'components/responsive';
+import * as styles from './card-pie-chart-styles.scss';
+import Tooltip from './tooltip/tooltip';
+
+class CardPieChart extends PureComponent {
+  renderAgricultureLabel = () => {
+    const { pieChartData } = this.props;
+    const color = pieChartData && pieChartData.color;
+    const emissionValue = pieChartData && pieChartData.emissionValue;
+    const emissionPercentage = pieChartData && pieChartData.emissionPercentage;
+
+    return (
+      <div className={styles.agricultureLabel}>
+        <div>
+          <Tag
+            key="agr"
+            label="Agriculture"
+            theme={{
+              tag: styles.tagTheme
+            }}
+            canRemove={false}
+            color={color}
+          />
+          <span className={styles.labelValue} style={{ color }}>
+            {`${emissionPercentage} (${emissionValue} `}
+            <span>
+              MtCO<sub>2</sub>
```

Comment:
we don't have that in the API?

Reply:
No, but maybe it is a good issue to address as a new backend task

Reply:
I think that because the unit is always the same we don't really have it on the backend, I don't think. So for now, probably OK if it's left like this.